### DISCUSSION TITLE
Why is the

### DISCUSSION THREAD

**Discussion by mkappworks** [supabase/supabase#27430] [Questions] [answered]
I am following this guide to create a supabase trigger on the auth.user table https://www.youtube.com/watch?v=0N6M5BBe9AE

But I am unable to select the auth.user from the `Table` drop down

![Screenshot 2024-06-20 at 22 34 18](https://github.com/supabase/supabase/assets/76746881/c1b69545-9e19-4656-acab-8c56f73fe996)

**Comment by GaryAustin1** [comment, marked as answer]
You have to add auth triggers with SQL in the SQL editor.

Something like:
```SQL

create trigger on_auth_user_created
  after insert on auth.users
  for each row execute procedure public.handle_new_user();
  ```
